example.config.json

```
@@ -0,0 +1,6 @@
+{
+  "auth_token":"AUTHTOKEN",
+  "dbip":"localhost",
+  "dbuser":"modbot",
+  "dbpassword":"password"
```

It's better to create objects for config categories like this:

```suggestion
  "db": {
      "ip": "localhost",
      "user": "modbot",
      "password": "password"
  }
```